pds_doi_service/core/actions/reserve.py

```diff
@@ -221,3 +243,6 @@ def run(self, **kwargs):
         # Convert all other errors into a CriticalDOIException to report back
         except Exception as err:
             raise CriticalDOIException(err)
+
+        # Return up-to-date version of output label(s)
+        return output_labels
```

Comment:
Double-check: in `release.py`, the return was `'\n'.join(output_labels)` as a `str` but here it's the list of `output_labels`—all good?